javascript/src/main/components/Statistics/AggregateStatisticsTable.js

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import BootstrapTable from 'react-bootstrap-table-next';
 
 const AgggregateStatisticsTable = ( {data} ) => {
-
+  console.log(data);
```

Comment:
We generally want to restrict console.log for debugging. Would we be able to remove this console.log?

Reply:
Issue fixed.